### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by tekishahammock**
Please review my material and provide me with feedback on how I can do better.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+	<link href='https://fonts.googleapis.com/css?family=Roboto:400,700,900,300' rel='stylesheet' type='text/css'>
+</head>
+<body>
+	<div class="container">
+
+		<!-- PRODUCT 1 ARTICLE -->
+
+		<div class="card">
```
Comment (NewEvolution) [inline index.html]: The instructions state each card should be an article, yours are each a div, _containing_ an article.  Close, but not quite.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+					<!-- three block elements that contain information about pricing -->
+					<p>1: $12.79</p>
+					<p>6: $84.23</p>
+					<p>Baker's Dozen: $193.56</p>
+				</section>
+			</article>
+		</div>
+	</div>
+</body>
+</html>
```
Comment (NewEvolution) [inline index.html]: As Github helpfully points out with the 🚫  icon, it's convention from þe olde POSIX days to end all files with a blank new line.  You can read more about it [here](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) if you're interested.

File: quiz.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.card-description p {
+  text-align: justify;
+  margin: 0 20px 20px;
+}
+
+.availability p{
+  color: yellow;
+  background-color: #5e5e5e;
```
Comment (NewEvolution) [inline quiz.css]: Believe it or not `darkgrey` is a real CSS color.  #5e5e5e is indeed dark grey, so no need to go and change this 😉

File: quiz.css [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.specs h3 {
+  font-weight: bold;
+  margin: 2px 10px 2px;
+}
+
+.specs p {
+  margin: 0 20px;
+}
```
Comment (NewEvolution) [inline quiz.css]: See above about the ending newline.

**Comment by tekishahammock** [comment]
Please re-review my quiz with the recommended changes and corrections.

**Comment by NewEvolution** [comment]
Looks great 👍

**Pull request merged**